### ISSUE TITLE
Remove PyDM Hack

### ISSUE THREAD

**Issue by slactjohnson** [pcdshub/opcpa-tpr-config#54]
Currently this code uses a version of PyDM from my dev directory. My dev area has a fix for a bug in PyDM that allows us to index the SEQCODES NTTable PV properly. 

Once the bug fix is merged into PyDM and released in a pcds environment, this should be removed.

**Comment by slactjohnson** [comment]
The first pcds-env with the PyDM bugfix implemented is pcds-6.0.0, with the actual latest release being 6.0.1. This environment is still in "open beta" so needs some testing.
